docs: small typos in single dc k8s clusters (#14484)
hashicorp/consul: website/content/docs/k8s/deployment-configurations/single-dc-multi-k8s.mdx

# Single Consul Datacenter in Multiple Kubernetes Clusters

> ~> **Note:** For running Consul across multiple Kubernetes clusters, it is generally recommended to utilize [Admin Partitions](/docs/enterprise/admin-partitions) for production environments. This Consul Enterprise feature allows for the ability to accommodate for multiple tenants without concerns of resource collisions when administering a cluster at scale, and for the ability to run Consul on Kubernetes clusters across a non-flat network. 

Reviewer: ```suggestion
~> **Note:** When running Consul across multiple Kubernetes clusters, we recommend using [admin partitions](/docs/enterprise/admin-partitions) for production environments. This Consul Enterprise feature allows you to accommodate multiple tenants without resource collisions when administering a cluster at scale. Admin partitions also enable you to run Consul on Kubernetes clusters across a non-flat network. 
```
A few adjustments to the rest of the text while we're at it
